redaxo/src/addons/cronjob/lib/manager.php

```diff
@@ -98,7 +98,7 @@ public function log($success, $message)
             ($success ? 'SUCCESS' : 'ERROR'),
             ($this->id ?: '--'),
             $name,
-            strip_tags($message),
+            strip_tags(nl2br($message),'<br><b>'),
```

Comment:
im logfile selbst html zu schreiben ist nicht sinnvoll. ggf. sollten wir die newlines schreiben und beim wiederauslesen dann in der weboberfläche `<br>` generieren..